lib/backends/sdc/utils.js

```diff
@@ -25,6 +25,38 @@ function vmUuidToShortDockerId(uuid) {
     return uuid.replace(/-/g, '').slice(0, 12);
 }
 
+function networkUuidToDockerId(uuid) {
+    return (uuid + uuid).replace(/-/g, '');
+}
+
+function shortNetworkIdToUuid(id) {
```

Comment:
I think that `shortNetworkIdToUuidPrefix` would be better, to make it clearer that it isn't converting to a full UUID.